tronweb2/src/components/JobsDashboard/JobsDashboard.js

```diff
@@ -58,7 +65,18 @@ function JobsDashboard() {
         </p>
       );
     } else {
-      jobContent = buildJobTable(jobData);
+      let jobsListToShow = jobData.jobs;
+      if (inputValue !== '') {
+        jobsListToShow = filterJobs(inputValue);
+      }
+      jobContent = (
+        <div>
+          <div className="mb-3">
+            <input type="text" className="form-control" placeholder="Search jobs" onInput={(e) => setInputValue(e.target.value)} />
```

Comment:
If we're going to keep it a prefix match can we change the placeholder?

Reply:
I'm using the Fuse library now, which should be much more powerful.